### ISSUE TITLE
Command line option of the pipeline

### ISSUE THREAD

**Issue by sekhwal** [phac-nml/snvphyl-galaxy#40]
Also, please let me know if I use the following command, does the pipeline cover all the steps from 1 to 5 of the workflow operations and generate the phylogeny tree. 
I am using the pipeline at Galaxy but it is taking very long time. I have total 70 isolates. 

$python bin/snvphyl.py --deploy-docker --fastq-dir example-data/fastqs --reference-file example-data/reference.fasta --min-coverage 5 --output-dir output1

**Comment by apetkau** [comment]
Yes, that command should cover all the steps. I would imagine it will take a long time with 70 isolates as it's got to spend a lot of time analyzing the genomes (aligning reads to a reference genome, identifying mutations and other variants, etc).

**Comment by sekhwal** [comment]
Hi,
I have a follow-up question regarding the pipeline.
Does the pipeline identify repeat regions on the reference genome using MUMMer automatically or should I perform MUMMer to identify repeats separately.

**Comment by apetkau** [comment]
@sekhwal yes, repeat region identification using MUMMer happens automatically in the below highlighted step in Galaxy:

![image](https://user-images.githubusercontent.com/200517/158409476-0de90f1a-d1d5-45a6-b8da-a334a881d102.png)
